go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/jrick/logrotate
+
+go 1.22
```

Comment:
The `github.com/klauspost/compress` library only seems to require `Go 1.20`. Could keep this at the same level to remove the need for also specifying `toolchain`...